examples/blog/src/layouts/BlogPost.astro

```diff
@@ -18,7 +18,8 @@ const {
 } = Astro.props;
 ---
 
-<html>
+<!DOCTYPE html>
+<html lang="en-us">
```

Comment:
My general instinct would be to default to something more general here:

```suggestion
<html lang="en">
```

Reply:
I agree, but the other templates have `en-us` and I was updating for consistency with those rather than introducing something different but still inconsistent. They could all be updated to `en`, but I felt that might have been overstepping.

Reply:
Fair! Ok with leaving this as `en-us` in that case 👍

Reply:
Ah, and now read your other comment! Thanks for looking at this in detail!